lang/python/python-ubus/Makefile

```
@@ -0,0 +1,40 @@
+#
+# Copyright (C) 2018-2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-ubus
+PKG_VERSION:=0.1.1
+PKG_RELEASE:=$(AUTORELEASE)
+
+PYPI_NAME:=ubus
+PKG_HASH:=7e57bda989bc35b48c7075d03ec2818226e722bbf1bde138d7e7ea26d462682a
+
+PKG_MAINTAINER:=Erik Larsson <who+openwrt@cnackers.org>
+PKG_LICENSE:=LGPLv2.1
```

This is not correct [SPDX Licence identifier](https://spdx.org/licenses/).

```suggestion
PKG_LICENSE:=LGPL-2.1-or-later
```

---

Fixed, but then the Makefile in turris-os-packages for python-ubus is incorrect

---

Yes, I agree on this one and that's not going to be fixed, because it will be removed there once this is merged here.